lib/offsite_payments/integrations/hi_trust.rb

```diff
@@ -31,7 +31,7 @@ def initialize(order, account, options = {})
           add_field('Type', 'Auth')
 
           # Capture the payment right away
-          add_field('depositflag', '1')
+          add_field('depositflag', '0')
```

Comment:
can we check `options` for `depositflag` before setting it? Just so that we can choose between the two if we need to